istio/metadata.csv

```diff
@@ -122,24 +122,24 @@ istio.pilot.duplicate_envoy_clusters,gauge,,,,Duplicate envoy clusters caused by
 istio.pilot.eds_no_instances,gauge,,,,Number of clusters without instances.,-1,istio,
 istio.pilot.endpoint_not_ready,gauge,,,,Endpoint found in unready state.,-1,istio,
 istio.pilot.invalid_out_listeners,gauge,,,,Number of invalid outbound listeners.,0,istio,
-istio.pilot.mcp_sink.reconnections,gauge,,,,The number of times the sink has reconnected.,0,istio,
-istio.pilot.mcp_sink.recv_failures_total,gauge,,,,The number of recv failures in the source.,-1,istio,
-istio.pilot.mcp_sink.request_acks_total,gauge,,,,The number of request acks received by the source.,0,istio,
+istio.pilot.mcp_sink.reconnections,count,,,,The number of times the sink has reconnected.,0,istio,
```

Comment:
is it ok to change these here?

Reply:
Yes, the metadata metric types are used as a default in the backend.